src/lib/eina/eina_inline_range.x

```diff
@@ -79,4 +79,5 @@ static inline Eina_Range eina_range_from_to(size_t min, size_t max)
    return EINA_RANGE(max, min - max);;
 }
 
-#endif // EINA_INLINE_RANGE_H_
\ No newline at end of file
+#endif // EINA_INLINE_RANGE_H_
+
```

Comment:
Why? XD

Reply:
haha this one is was a fix for a repetitive warning about not having a proper line ending: b9643cd